I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 44, in script
    f "You can try to flee from the chill. But the cold's in your bones now."
Exception: The say screen (or show_function) must return a Text object.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 44, in script
    f "You can try to flee from the chill. But the cold's in your bones now."
  File "/Users/nichosantos/Documents/[CODING]/renpy-8.2.3-sdk/renpy/ast.py", line 2586, in execute
    Say.execute(self)
  File "/Users/nichosantos/Documents/[CODING]/renpy-8.2.3-sdk/renpy/ast.py", line 623, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "/Users/nichosantos/Documents/[CODING]/renpy-8.2.3-sdk/renpy/exports/sayexports.py", line 132, in say
    who(what, *args, **kwargs)
  File "/Users/nichosantos/Documents/[CODING]/renpy-8.2.3-sdk/renpy/character.py", line 1471, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "/Users/nichosantos/Documents/[CODING]/renpy-8.2.3-sdk/renpy/character.py", line 1117, in do_display
    display_say(who,
  File "/Users/nichosantos/Documents/[CODING]/renpy-8.2.3-sdk/renpy/character.py", line 764, in display_say
    raise Exception("The say screen (or show_function) must return a Text object.")
Exception: The say screen (or show_function) must return a Text object.

macOS-13.4-arm64-arm-64bit arm64
Ren'Py 8.3.7.25031702
PLEASE IM BEGGING YOU 1.0
Mon May  5 00:30:27 2025
